docs/extensions/index.md

```diff
@@ -13,7 +13,9 @@ This section covers how to write extensions.
 
 ## Extension Names
 
-All extension names follow the same convention as Java package names – they look like reversed domain names. (e.g.  `com.example.myextension`). For module-extensions, the last word in the module name will be the module's *short-name*. The short-name *must* be unique. It is possible to pick a different short-name, but that requires extra work.
+Pick a unique single word for your extension's name. Note that this name is used throughout the extension's code, e.g. function names so it should only include characters that can be safely used in that way.
```

Comment:
```suggestion
Pick a unique string for your extension's name. Note that this name is used throughout the extension's code, e.g. in function names so it should only include characters that can be safely used in that way.

!!! warning
    There is currently no "canonical list" of extension names. We'd strongly recommend you check the [extensions directory](https://civicrm.org/extensions/) and the [GitLab extensions group](https://lab.civicrm.org/extensions).
```

Reply:
Those are good questions @totten @mlutfy can you provide clarity with regards the policy used for GitLab namespaces and in-app limitations?

Reply:
Minor readability nitpick: on line 16, could you add a comma beteween "names" and "so"? Then it would read "... code, e.g. in function names, so ..."

Reply:
> how you edit things ↑ here like @MikeyMJCO did.

Yeah, I was wondering the same thing.

Reply:
> > how you edit things ↑ here like @MikeyMJCO did.
> 
> Yeah, I was wondering the same thing.

In "Review"/"Files Changed" when you over over a line you'll get a + button - you can then click the little "Insert a Suggestion" button (CTRL+G)

Reply:
```suggestion
Pick a unique single word for your extension's name. Note that this name is used throughout the extension's code, e.g. function names, so it should only include characters that can be safely used in that way.
```